edx_rest_framework_extensions/auth/jwt/tests/test_middleware.py

```diff
@@ -349,12 +355,22 @@ def test_login_required_middleware(self, url, has_jwt_cookies, expected_status):
     def test_login_required_overridden_middleware(self, url, has_jwt_cookies, expected_status):
         if has_jwt_cookies:
             self.client.cookies = _get_test_cookie()
+            if get_setting(ENABLE_FORGIVING_JWT_COOKIES):
+                expected_status = 200
         response = self.client.get(url)
         self.assertEqual(expected_status, response.status_code)
         if response.status_code == 302:
             self.assertEqual('/overridden/login/?next=' + url, response.url)
 
 
+# We want to duplicate these tests for now while we have two major code paths.  It will get unified once we have a
+# single way of doing JWT authentication again.
+@ddt.ddt
+@override_settings(EDX_DRF_EXTENSIONS={ENABLE_FORGIVING_JWT_COOKIES: True})
+class TestForgiingJwtRedirectToLoginIfUnauthenticatedMiddleware(TestJwtRedirectToLoginIfUnauthenticatedMiddleware):  # pylint: disable=test-inherits-tests # noqa E501 line too long
```

Comment:
1. Typo:
```suggestion
class TestForgivingJwtRedirectToLoginIfUnauthenticatedMiddleware(TestJwtRedirectToLoginIfUnauthenticatedMiddleware):  # pylint: disable=test-inherits-tests # noqa E501 line too long
```
2. I think I don't see any of the same issues that I saw with the other duplicated test class, but not certain.